apps/crn-server/test/data-providers/contentful/tutorials.data-provider.test.ts

```diff
@@ -19,32 +19,45 @@ describe('Tutorials data provider', () => {
   const dataProvider: TutorialDataProvider = new TutorialContentfulDataProvider(
     contentfulGraphqlClientMock,
   );
-
   const graphqlTutorialResponse = getContentfulGraphqlTutorial();
-  const contentfulGraphqlClientMockServer =
-    getContentfulGraphqlClientMockServer({
-      Discover: () => getContentfulGraphqlDiscover(),
-      DiscoverTrainingCollection: () => {
-        return {
-          total: 1,
-          items: [graphqlTutorialResponse],
-        };
-      },
-      Tutorials: () => graphqlTutorialResponse,
-      TutorialsCollection: () => {
-        return graphqlTutorialResponse.linkedFrom?.tutorialsCollection;
-      },
-      RelatedTutorialsCollection: () => {
-        return graphqlTutorialResponse.relatedTutorialsCollection;
-      },
-    });
-  const dataProviderWithMockServer: TutorialDataProvider =
-    new TutorialContentfulDataProvider(contentfulGraphqlClientMockServer);
 
   describe('Fetch', () => {
+    let tutorialsCollectionHitCount = 0;
+    beforeEach(() => {
+      tutorialsCollectionHitCount = 0;
+    });
+    const contentfulGraphqlClientMockServer =
+      getContentfulGraphqlClientMockServer({
+        Discover: () => getContentfulGraphqlDiscover(),
+        DiscoverTrainingCollection: () => {
+          return {
+            total: 1,
+            items: [graphqlTutorialResponse],
+          };
+        },
+        Tutorials: () => graphqlTutorialResponse,
+        TutorialsCollection: () => {
+          // Due to querying tutorial linked references this resolver gets hit twice
+          // packages/contentful/src/crn/queries/tutorials.queries.ts
+          if (tutorialsCollectionHitCount === 0) {
```

Comment:
what would happen if you didn't conditionally return this and just used a normal fixture instead?